django/db/models/base.py

```diff
@@ -244,13 +244,21 @@ def __new__(cls, name, bases, attrs):
                     field = None
                 new_class._meta.parents[base] = field
```

Comment:
Not actually related, I'll add an explicit `parent_link=True` to the `OneToOneField` to make it more obvious.